### PR TITLE
Init UI for showing search history

### PR DESCRIPTION
**Pull request by thaohq-0814**
Init UI for showing search history
Link: https://edu-redmine.framgia.vn/issues/6955
image

### CODE REVIEW

**Comment by thaohq-0814** [comment]
ready

File: app/src/main/res/layout/item_search_history.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            type="com.framgia.my_editor_02.screen.search.ItemHistoryViewModel"
+            />
+    </data>
+    <android.support.constraint.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:onClick="@{viewModel::onItemClicked}"
+        >
+        <TextView
+            android:id="@+id/textView"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/item_search_history.xml]: check convention

File: app/src/main/java/com/framgia/my_editor_02/screen/main/MainActivity.java [hunk 2]
```diff
@@ -34,4 +41,22 @@ protected void onStop() {
         super.onStop();
         mMainViewModel.onStop();
     }
+
+    @Override
+    public boolean dispatchTouchEvent(MotionEvent event) {
+        if (event.getAction() == MotionEvent.ACTION_DOWN) {
+            View v = getCurrentFocus();
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/my_editor_02/screen/main/MainActivity.java]:
```suggestion
            View view = getCurrentFocus();
```

**Comment by thaohq-0814** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**